styles/css/themes/light/abstraction-variables.css

```diff
@@ -0,0 +1,301 @@
+:root {
+  --pgn-elevation-box-shadow-level-1:
```

Comment:
[nit/suggestion] I might structure this file with comments to split out the different "groups" of variables to make the file more parseable/readable.

For example, all the `elevation` related tokens could be grouped under a section heading like:

```css
/**
 * ELEVATION
 */
```

or something similar :)

Reply:
Added, good idea 💯